src/write_gcm.c

```diff
@@ -3869,13 +3869,13 @@ static int wg_transmit_unique_segment(const wg_context_t *ctx,
 
       wg_log_json_message(
           ctx, "Server response (CollectdTimeseriesRequest):\n%s\n", response);
-      // Since the response is expected to be valid JSON, we don't
-      // look at the characters beyond the closing brace.
+      // The response is expected to be valid JSON. In case the response
+      // isn't empty, a potential partial success case may have occured. In
+      // this case we increment the API error count but don't flush the
```

Comment:
This comment is no longer accurate. Also below.

Reply:
Fixed. PTAL